rina-tools/src/rlite/iporinad.cpp

```diff
@@ -1227,23 +1221,55 @@ IPoRINA::connect_to_remotes()
     }
 }
 
+/* Turn this program into a daemon process. */
+static void
+daemonize(void)
+{
+    pid_t pid = fork();
+    pid_t sid;
+
+    if (pid < 0) {
+        perror("fork(daemonize)");
+        exit(EXIT_FAILURE);
+    }
+
+    if (pid > 0) {
+        /* This is the parent. We can terminate it. */
+        exit(0);
+    }
+
+    /* Execution continues only in the child's context. */
+    sid = setsid();
+    if (sid < 0) {
+        exit(EXIT_FAILURE);
+    }
+
+    chdir("/");
+}
+
 static void
 usage(void)
 {
     cout << "iporinad [OPTIONS]" << endl
          << "   -h : show this help" << endl
          << "   -c CONF_FILE: path to config file "
             "(default /etc/rina/iporinad.conf)"
-         << endl;
```

Comment:
This endl should be there, otherwise the help option produces this output:

````
iporinad [OPTIONS]
   -h : show this help
   -c CONF_FILE: path to config file (default /etc/rina/iporinad.conf)   -L NUM : maximum loss probability introduced by the flow (NUM/10000)
   -E NUM : maximum delay introduced by the flow (microseconds)
   -v : be verbose
````

Reply:
should be fixed now